truss/templates/trtllm-briton/src/engine.py

```diff
@@ -224,11 +224,15 @@ async def predict(self, model_input):
         ):
             request.pad_id = self._tokenizer.pad_token_id
         # Add output schema hash if we're function calling or response_format is provided
-        schema_hash = (
-            self._fsm_cache.add_schema(function_calling_schema)
-            if function_calling_schema is not None
-            else self._fsm_cache.add_schema_from_input(model_input)
-        )
+        schema_hash = None
+        try:
+            schema_hash = (
+                self._fsm_cache.add_schema(function_calling_schema)
+                if function_calling_schema is not None
+                else self._fsm_cache.add_schema_from_input(model_input)
+            )
+        except NotImplementedError as ex:
```

Comment:
Worth adding a comment here as to why this exception may be thrown. Great that we're handing a particular exception.